jso/apis/src/main/java/org/teavm/jso/dom/events/WheelEvent.java

```diff
@@ -0,0 +1,35 @@
+/*
+ *  Copyright 2014 Alexey Andreev.
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+package org.teavm.jso.dom.events;
+
+import org.teavm.jso.JSProperty;
+
+/**
+ *
+ */
+public interface WheelEvent extends MouseEvent {
+    @JSProperty
+    double getDeltaX();
+
+    @JSProperty
+    double getDeltaY();
+
+    @JSProperty
+    double getDeltaZ();
+
+    @JSProperty
+    long getDeltaMode();
```

Comment:
This won't work in JSO, since it does not allow to pass `long` to and from JS. Please, use `int` instead. Also, providing constants would be useful.